apps/src/templates/courseOverview/CourseOverview.js

```diff
@@ -98,7 +101,7 @@ export default class CourseOverview extends Component {
               onChange={this.onChangeVersion}
               value={name}
               style={styles.versionDropdown}
-              className="version-selector"
+              id="version-selector"
```

Comment:
is this for UI tests only? I've seen us do `uitest-version-selector` to indicate its for testing so it doesn't get removed by accident.

Reply:
nope, this one is for both ui tests and unit tests. 

going forward, I do not understand the rationale for this convention -- by definition a test which depends on this kind of id or classname will prevent it from being removed. by contrast a classname which only affects style/appearance is much more likely to break things and go unnoticed, unless an eyes test happens to be in place. so I'm inclined to relax this convention unless there is a reason not to. if there is a benefit i'm missing please let me know :-)